trade-finance-manager-api/src/v1/controllers/user/passport.js

```diff
@@ -33,6 +33,15 @@ module.exports = (passport) => {
         return done(error, false);
       }
 
+      if (!user) {
+        console.error('User is not found in DB');
+      }
+      if (user && user.sessionIdentifier !== jwtPayload.sessionIdentifier) {
+        console.error("SESSION ID doesn't match");
+        console.error('DB user.sessionIdentifier', user.sessionIdentifier);
+        console.error('REQ jwtPayload.sessionIdentifier', jwtPayload.sessionIdentifier);
+      }
+
```

Comment:
Same as above, maybe could be e.g

```js
console.error("Error finding user by username - SESSION ID doesn't match");
```

Reply:
I added this logging for my testing, not sure if we should keep it. PR is Work in progress :) .

Happy to improve text and leave one `console.error` call.